pymc3/blocking.py

```diff
@@ -67,45 +67,52 @@ class DictToArrayBijection:
     A mapping between a dict space and an array space
     """
 
-    def __init__(self, ordering, dpoint):
+    def __init__(self, ordering: List[Text]):
+        # TODO: Should just use `OrderedDict`s and remove this state entirely
```

Comment:
It is, but I didn't want to make that assumption unless it was already an assumption of the project itself (e.g. it only supports Python versions for which that's necessarily true).

On a related note, the `self.ordering`, and perhaps the entire `DictToArrayBijection` class is unnecessary with ordered `dict`s.  Since the `vars` in a `Model` already have an order, and we create these `dict`s using `vars`, this entire class can be replaced by two simple helper functions derived from `DictToArrayBijection.map` and `DictToArrayBijection.rmap`.

Reply:
https://github.com/pymc-devs/pymc3/blob/e467bb91890f8a6707a8e4e0ebf01af93f1e7145/setup.py#L85

Reply:
As of 3.7, they are ordered.

Reply:
All right, then I'll simplify this by using the `dict` order itself.